src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -805,31 +795,26 @@ namespace stan {
     validate_void_return_allowed_f;
 
     void identifier_to_var::operator()(const std::string& name,
-                                       const var_origin& origin,
+                                       const scope& var_scope,
                                        variable& v,  bool& pass,
                                        const variable_map& vm,
                                        std::ostream& error_msgs) const {
       // validate existence
       if (!vm.exists(name)) {
-        pass = false;
+        pass = false;   // MM: no error message?  see validate_assignment below
```

Comment:
Inline comment style is `// TODO(morris): yada yada yada`

Would you mind checking through what happens when that identifier to var doesn't pass?  I think we could squeeze in an error message here.  Feel free to open a new issue for the resolution.